internal/ansible/controller/reconcile.go

```diff
@@ -170,9 +170,21 @@ func (r *AnsibleOperatorReconciler) Reconcile(ctx context.Context, request recon
 	// iterate events from ansible, looking for the final one
 	statusEvent := eventapi.StatusJobEvent{}
 	failureMessages := eventapi.FailureMessages{}
+	prevEvent := ""
 	for event := range result.Events() {
 		for _, eHandler := range r.EventHandlers {
+			if event.Event == eventapi.EventRunnerItemOnOk && prevEvent != eventapi.EventRunnerItemOnOk { // Loop begins
+				fmt.Printf("\n--------------------------- Ansible Task StdOut -------------------------------\n")
```

Comment:
Let's keep all the actual logging in the event logging package

Reply:
Yeah, I thought the same. But according to this loop, each iteration of the ansible loop gets a separate call to the `Handle` function. However, the handle function (`log_events.go`), requires us to print out the event in this format:
```
--------------------------- Ansible Task StdOut -------------------------------

 TASK [Debug with loop] ********************************

-------------------------------------------------------------------------------
```
And so the output would become
```
--------------------------- Ansible Task StdOut -------------------------------

 TASK [Debug with loop] ********************************
ok: [localhost] => (item=1) => {
    "msg": 1
}

-------------------------------------------------------------------------------

--------------------------- Ansible Task StdOut -------------------------------

 TASK [Debug with loop] ********************************
ok: [localhost] => (item=2) => {
    "msg": 2
}

-------------------------------------------------------------------------------
```

Which is not what we want, right? Is there any other workaround?

Reply:
Okay, I moved all this logic to `log_events.go` by passing an additional parameter.